Makefile

```diff
@@ -183,10 +184,11 @@ doc:
 doc-spelling:
 	@make -C docs spelling SPHINXOPTS="-W --keep-going -n -E"
 
+requirements/%.txt: requirements/%.in $(filter-out $(REQIN2),$(REQS))
+	pip-compile --no-header --allow-unsafe -q -o $@ $(@:.txt=.in)
```

Comment:
Why `--no-header`?

Reply:
`header` contains Python version, files generated by Python 3.9 based environment are different from 3.10 for example.

Looks weird.